mlflow/pyfunc/__init__.py

```diff
@@ -2236,6 +2247,11 @@ def predict(model_input: List[str]) -> List[str]:
     _validate_env_arguments(conda_env, pip_requirements, extra_pip_requirements)
     _validate_pyfunc_model_config(model_config)
     if python_model:
+        model_code_path = None
+        if isinstance(python_model, str):
+            model_code_path = _validate_and_get_model_code_path(python_model)
+            python_model = _load_model_code_path(model_code_path)
```

Comment:
Should we call `_validate_and_copy_model_code_and_config_paths` before loading or else I am not sure how this will work

Reply:
🤦‍♂️ sorry